raglweb/__init__.py

```diff
@@ -198,7 +204,18 @@ def playoffs():
 def games():
     db = _db_get()
     games = _get_games(db, 'outcomes')
-    return render_template('games.html', games=games)
+
+    # create an unspoiled list of games (players ordered alphanumerically, additional "winner" attribute)
+    unspoiled_games = []
+    for game in games:
+        p0, p0_id, p1, p1_id = game['p0'], game['p0_id'], game['p1'], game['p1_id']
+        # re-order players alphanumerically if necessary
+        if not _ordered_alphanumerically(p0, p1):
+            game.update(p0=p1, p0_id=p1_id, p1=p0, p1_id=p0_id)
+        game['winner'] = p0
+        unspoiled_games.append(game)
```

Comment:
Now that I think again, do you even need the `unspoiled_games` array? I believe the loop is mutating the values of `games` in the loop anyway so it shouldn't be needed.

Reply:
This also worked, well spotted :+1: